static/js/classroom/game.js

```
@@ -0,0 +1,566 @@
+// Handle player movement with animations
+function handlePlayerMovement() {
+  if (!cursors) return;
+
+  const speed = 160;
+  let velocityX = 0;
+  let velocityY = 0;
+  let animation = 'idle';
+  let direction = 'down';
+
+  // Get current direction from animation if playing
+  if (player.anims && player.anims.isPlaying) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)